### PR TITLE
Feature/descw 1004 pattern cpt functionality updates

### PR DESCRIPTION
**Pull request by robrien-bcgov**
- prefix child pattern groups with parent names in patterns selector ([DESCW-1004](https://apps.itsm.gov.bc.ca/jira/browse/DESCW-1004))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.2.6 April 17, 2023
```
Comment (ShawnTurple) [inline CHANGELOG.md]: 1.2.0 not 1.2.6

File: composer.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "bcgov-theme/bcgov-wordpress-block-theme",
     "description": "Block Theme for BC Gov",
-    "version": "1.1.6",
+    "version": "1.2.6",
```
Comment (ShawnTurple) [inline composer.json]:
```suggestion
    "version": "1.2.0",
```

File: CHANGELOG.md (hunk 1 above)
Comment (Baelx) [inline CHANGELOG.md]:
```suggestion
## 1.2.0 April 17, 2023
```

Usually with semver, you bump to the next minor version and set the patch level to 0.

**Review by Baelx — CHANGES_REQUESTED** [review]
NW: Looks good! I think the version number just needs to be changed to be in accordance with semver

File: src/Actions/PatternsSetup.php [hunk 3]
```diff
@@ -222,13 +229,35 @@ public function bcgov_blocks_theme_register_block_patterns() {
...
+						}
+
 						if ( ! \WP_Block_Patterns_Registry::get_instance()->is_registered( 'bcgov_blocks_theme-' . $category->slug ) ) {
 							register_block_pattern_category(
-								'bcgov_blocks_theme-' . $category->slug,
+                                'bcgov_blocks_theme-' . $category->slug,
 								[
 									/* translators: %s: category label */
-									'label' => sprintf( __( 'Custom: %s', 'bcgov_blocks_theme' ), $category->name ),
+									'label' => sprintf( __( '%1$s %2$s', 'bcgov_blocks_theme' ), $parent_name, $category->name ),
```
Comment (ShawnTurple) [inline src/Actions/PatternsSetup.php]: This means that all the non parent items will always have a whitespace, better to add the whitespace at end of parent name
```suggestion
									'label' => sprintf( __( '%1$s%2$s', 'bcgov_blocks_theme' ), $parent_name, $category->name ),
```

File: style.css [hunk 4]
```diff
@@ -7,7 +7,7 @@ Description: WordPress Block theme is a theme that leverages the full-site editi
 Requires at least: 5.9
 Tested up to: 6.1
 Requires PHP: 7.4
-Version: 1.1.6
+Version: 1.2.6
```
Comment (ShawnTurple) [inline style.css]:
```suggestion
Version: 1.2.0
```

File: CHANGELOG.md (hunk 1 above)
Comment (robrien-bcgov) [reply CHANGELOG.md]: done. ready for re-review

**Review by ShawnTurple — CHANGES_REQUESTED** [review]
Fix whitespace issue with regards to pattern category
Bonus: look at get_term_parents_list to see if you can clean up parent code.

File: src/Actions/PatternsSetup.php (hunk 3 above)
Comment (robrien-bcgov) [reply src/Actions/PatternsSetup.php]: done

File: composer.json (hunk 2 above)
Comment (robrien-bcgov) [reply composer.json]: done

**Review by Baelx — APPROVED** [review]
Thanks! Approved

**Review by ShawnTurple — APPROVED** [review]

**Pull request merged**